### ISSUE TITLE
dict(zip(keys, values)) not correctly inferred when values is a tuple of operators

### ISSUE THREAD

**Issue by twoertwein** [microsoft/pyright#6695]
**Describe the bug**
`dict(zip(keys, values))` is inferred incorrectly when `values` is a tuple of operators (the type of `keys` and `values` are being inferred correctly).

**Code or Screenshots**
```python
# pyright: strict

import operator

# correct (keys are str)
keys = ("+",)
values = (operator.pos,)
mapping = dict(zip(keys, values))
reveal_type(mapping)

# incorrect (keys and values are bytes?)
keys = ("+", "-")
values = (operator.pos, operator.neg)
mapping = dict(zip(keys, values))
reveal_type(mapping)
```

**VS Code extension or command-line**
Pyright (1.1.339) from the command-line

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.340, which I just published. It will be included in a future release of pylance.
